### DISCUSSION TITLE
using CalcLigRMSD without GetPDBResidueInfo

### DISCUSSION THREAD

**Discussion by YoavShamir5** [rdkit/rdkit#7828] [Q&A]
Hello, I tried to use CalcLigRMSD (from the Contrib dir) with molecules from sdf files, but the following line in CalcLigRMSD.py led to error, since the atoms do not have GetPDBResidueInfo data:

**atom_names_lig1 = [atom1.GetPDBResidueInfo().GetName() for atom1 in lig1.GetAtoms()]**

Is there a workaround to use CalcLigRMSD in such a case? Can I generate GetPDBResidueInfo for atoms on-the-fly just for CalcLigRMSD? If not - does RDKit have another way to calculate RMSD for two ligands without aligning them, while taking symmetry into consideration?

**Comment by cespos** [comment]
`GetPDBResidueInfo` is only called when `rename_lig2 = True`. However, if you do not need to rename lig2 based on the atom names of lig1, `CalcLigRMSD` shoud work.
